lib/Remotery.c

```diff
@@ -2366,12 +2373,19 @@ typedef struct
 
     rmtU32 frame_bytes_remaining;
     rmtU32 mask_offset;
+#ifdef _MSC_VER
+    pragma warning(push)
 
+    #pragma warning(disable:4201) // Nameless struct/union
+#endif
     union
     {
         rmtU8 data_mask[4];
         rmtU32 data_mask_u32;
     };
+#ifdef _MSC_VER
+    #pragma warning(pop)
+#endif
 } WebSocket;
 
```

Comment:
Yeah, that's exactly what I meant. Name the union `data` and then all variable references change to `data.mask` and `data.mask_u32`. Breaking changes within the .c file are fine.